app/components/finance/npq/participant_outcomes/table.rb

```diff
@@ -16,16 +14,62 @@ def render?
           participant_declaration.participant_profile.npq? && outcomes.present?
         end
 
+        def state(outcome)
+          outcome.state&.capitalize
+        end
+
         def completion_date(outcome)
           outcome.completion_date.to_fs(:govuk)
         end
 
         def changed_date(outcome)
-          outcome.created_at.to_fs(:govuk)
+          outcome.created_at.to_date.to_fs(:govuk)
+        end
+
+        def sent_to_tra_at(outcome)
+          return "<strong class='govuk-tag govuk-tag--yellow'>#{t('finance.npq.participant_outcomes.na')}</strong>".html_safe if outcome.sent_to_qualified_teachers_api_at.blank?
+
+          outcome.sent_to_qualified_teachers_api_at.to_date.to_fs(:govuk)
+        end
+
+        def sent_to_tra_tag(bool)
+          if bool.nil?
+            "<strong class='govuk-tag govuk-tag--yellow'>#{t('finance.npq.participant_outcomes.na')}</strong>"
+          elsif bool
+            "<strong class='govuk-tag govuk-tag--green'>#{t('finance.npq.participant_outcomes.yes')}</strong>"
+          else
+            "<strong class='govuk-tag govuk-tag--red'>#{t('finance.npq.participant_outcomes.no')}</strong>"
+          end.html_safe
         end
 
-        def sent_to_tra(outcome)
-          outcome.sent_to_qualified_teachers_api_at&.to_fs(:govuk)
+        def caption_text
+          titles = [t("finance.npq.participant_outcomes.declaration_outcomes")]
```

Comment:
The use of an array here initially made me think there could be multiple titles... could just be me, but it may be clearer without it 🤷‍♂️ 

```
def caption_text
  title = t("finance.npq.participant_outcomes.declaration_outcomes")
  latest_outcome = outcomes.first

  return title unless latest_outcome.present?
  
  if latest_outcome.passed_and_not_sent?
    outcome_description = t("finance.npq.participant_outcomes.passed")
  elsif latest_outcome.failed_and_not_sent?
    outcome_description = t("finance.npq.participant_outcomes.failed")
  elsif latest_outcome.passed_and_recorded?
    outcome_description = t("finance.npq.participant_outcomes.passed_and_recorded")
  elsif latest_outcome.failed_and_recorded?
    outcome_description = t("finance.npq.participant_outcomes.failed_and_recorded")
  elsif latest_outcome.passed_and_not_recorded?
    outcome_description = t("finance.npq.participant_outcomes.passed_and_not_recorded")
  elsif latest_outcome.failed_and_not_recorded?
    outcome_description = t("finance.npq.participant_outcomes.failed_and_not_recorded")
  end

  "#{title}: #{outcome_description}"
end
```